events/dvg_dua.txt

```diff
@@ -0,0 +1,3882 @@
+				trigger = {
+					has_journal_entry = je_english_integration
+				}
+			}
+		}
+	}
+
+	event_image = {
+		video = "gfx/event_pictures/southamerica_aristocrats.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/southamerica/aristocrats"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+	
+	immediate = {
+		if = {
+			limit = {
+				NOT = { exists = c:OCC }
+			}
+
+			create_country = {
+				tag = OCC
+				origin = this
+				state = s:STATE_ARMAGNAC.region_state:DUA
+				on_created = {
+					add_modifier = {
+						name = dvg_revolutionary_militias
+						days = 720
+					}
+					add_loyalists = {
+						value = 0.25
+						culture = cu:occitan
+					}
+				}
+			}
+
+			create_diplomatic_play = {
+				target_country = c:OCC
+				war = no
+				type = dp_rebellion
+			}
+		}
+
+		every_scope_state = {
+			limit = {
+				owner = c:DUA
+				state_region = {
+					is_homeland = cu:occitan
+				}
+			}
+			set_state_owner = c:OCC
+		}
+		
+		if = {
+			limit = { exists = s:STATE_SAVOY.region_state:DUA }
+			s:STATE_SAVOY.region_state:DUA = { set_state_owner = c:OCC }
+		}
+
+		if = {
+			limit = {
+				has_journal_entry = je_dual_integration
+			}
+			c:OCC = {
+				set_variable = dual_integration_secession
+			}
+		}
+		else_if = {
+			limit = {
+				has_journal_entry = je_english_integration
+			}
+			c:OCC = {
+				set_variable = english_integration_secession
+			}
+		}
+	}
+
+	option = {
+		name = dvg_dua.510.a
+		default_option = yes
+
+		c:OCC = {
+			trigger_event = {
+				id = dvg_dua.511
+				days = 0
+				popup = yes
+			}
+		}
+	}
+}
+
+dvg_dua.511 = { #Midi Revolution - Occitan choices
+	type = country_event
+	placement = ROOT
+
+	title = dvg_dua.511.t
+	desc = dvg_dua.511.d
+	flavor = dvg_dua.511.f
+
+	event_image = {
+		video = "gfx/event_pictures/southamerica_aristocrats.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/southamerica/aristocrats"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+
+	trigger = {
+	}
+
+	immediate = {
+
+	}
+
+	option = {
+		name = dvg_dua.511.a
+		default_option = yes
+
+		trigger = {
+			exists = c:PRO
+		}
+
+		c:PRO = {
+			trigger_event = {
+				id = dvg_dua.512
+				days = 1
+				popup = yes
+			}
+		}
+
+		ai_chance = {
+			base = 75
+		}
+	}
+
+	option = {
+		name = dvg_dua.511.b
+
+		ig:ig_landowners = { 
+			remove_ideology = ideology_paternalistic
+			add_ideology = ideology_republican_paternalistic
+		}
+
+		if = {
+			limit = {
+				NOT = { has_law = law_type:law_parliamentary_republic }
+			}
+			activate_law = law_type:law_presidential_republic
+		}
+		if = {
+			limit = {
+				NOT = { has_law = law_type:law_right_of_assembly }
+			}
+			activate_law = law_type:law_right_of_assembly
+		}
+
+		ai_chance = {
+			base = 25
+		}
+	}
+}
+
+dvg_dua.512 = { #Midi Revolution - PRO inherits OCC
```

Comment:
So for the Midi revolution reaction of Provence, you should add the ability of Provence to decline
Just for a player provence, can have AI provence 100% accept, and the AI makes event decisions immediately, meaning there isn't a gap between doing it this and using the immediate section 
It is one of those illusion of choice deals, and I think it adds a bit to the experience and makes it feel like the player has more control 
"Alright, I'll switch over and become big provence"
as opposed to
"Alright... Damn it I just wanted to sit here and build up Provence, maybe go into Italy(once that is added), but now I have all of occitania and the Dual monarchy is about to annex me? And I just, didn't get a choice?"

Reply:
OC, I definitely should mention i think in almost every case player will choose big Provence, just think that, as I said, the option is there if someone does want to decline, for whatever reason.